### PR TITLE
Julia 1.8.5 upgrade

### PR DESCRIPTION
**Pull request by nikishe**
Julia docs and install script , updated to cover version 1.8.5

### CODE REVIEW

**Comment by jkwmoore** [comment]
LGTM, we may want to flesh this page out with a bit more info in future though. Examples etc...

File: sharc/software/apps/julia.rst [hunk 1]
```diff
@@ -27,6 +28,10 @@ Installation notes
 ------------------
 These are primarily for administrators of the system.
 
+**julia version 1.8.5**
+was installed using the
+:download:`install_julia.sh </sharc/software/install_scripts/apps/julia/1.8.5/install_julia.sge>` script.
```
Comment (jkwmoore) [inline sharc/software/apps/julia.rst:33]: At present the file has the .sge file extension so we should either match the extension here or change the file extension to .sh ?

Comment (willfurnass) [reply sharc/software/apps/julia.rst:33]: I suggest we address general refinements to this page (and ideally the Bessemer one for parity) as a separate Issue / PR.

**Comment by willfurnass** [comment]
LGTM but would be good to commit the 1.8.5 modulefile under `sharc/software/modulefiles/apps/julia/`

**Comment by nikishe** [comment]
Made necessary changes and added the module file

File: sharc/software/apps/julia.rst (hunk 1 above)
Comment (nikishe) [reply sharc/software/apps/julia.rst]: updated

**Review by willfurnass — APPROVED** [review]

**Comment by willfurnass** [comment]
@nikishe Once the PR has been reviewed and approved, feel free to merge into `master`.

**Pull request merged**